patroni/__init__.py

```diff
@@ -129,7 +129,10 @@ def setup_signal_handlers(self):
         signal.signal(signal.SIGTERM, self.sigterm_handler)
 
     def shutdown(self):
-        self.api.shutdown()
+        try:
+            self.api.shutdown()
+        except Exception:
+            logger.exception('Exception durig RestApi.shutdown')
```

Comment:
during.

Reply:
also want to see the specifics on the exception

Reply:
what specifics?
logger.exception will show the full stacktrace